config/jobs/periodic/docker-in-docker/periodic-dind-build.yaml

```diff
@@ -0,0 +1,213 @@
+periodics:
+  - name: docker-build
+    labels:
+      preset-secrets: "true"
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    interval: 24h
+    spec:
+      containers:
+      - image: quay.io/florencepascual/docker_ce_build
+        command:
+        - /bin/bash
+        args:
+        - -c
+        - |
+          # path to the github repository
+          DIR_GITHUB="docker-ce-build"
+          PATH_GITHUB="https://github.com/florencepascual/${DIR_GITHUB}.git"
+          
+          # path to the image for building and testing
+          PATH_IMAGE_BUILD="quay.io/florencepascual/docker_ce_build"
+
+          # path to the scripts 
+          PATH_SCRIPTS="/workspace/${DIR_GITHUB}"
+          DATE=`date +%d%m%y-%H%S`
+          PATH_LOG_PROWJOB="/workspace/prow-job-${DATE}.log"
+
+          export DIR_GITHUB
+          export PATH_SCRIPTS
+          export PATH_LOG_PROWJOB
+
+          # Create the log file
+          echo "Create the prow job log file"
+          touch ${PATH_LOG_PROWJOB}
```

Comment:
Do we explicitly need to create the log file with the touch command and check its successful creation?
I see bash creating and appending data to file when we use `tee -a`

Reply:
Actually, we do not need to check it, I will remove that